README.md

```diff
@@ -144,3 +147,329 @@ Otherwise you will need to tell the VM each time as well like so:
+```
+Sample  Chromosome      Start   End     Num_Probes      Segment_Mean
+SAMPLE1        1       12200   70000   18       0.841235
+SAMPLE1        1       300600  1630000 337     1.23232323
+....snip....
+```
+
+###### Invocation
+
+```
+java -Xmx8g -jar <path_to_hellbender_jar>  PerformCBSSegmentation  -S <sample_name> -T <target_file_tsv> -O <output_seg_file> -log
+```
+
+
+##### Step 4. Call segments
+
+###### Inputs
+- target file (tsv) -- from step 2.
+- seg file (tsv) -- from step 3.
+- sample name
+
+###### Outputs
+- called file (tsv) -- output is exactly the same as in seg file (step 3), except Segment_Call column is added.  Calls are either "+", "0", or "-" (no quotes). 
+```
+Sample  Chromosome      Start   End     Num_Probes      Segment_Mean      Segment_Call
+SAMPLE1        1       12200   70000   18       0.841235      -
+SAMPLE1        1       300600  1630000 337     1.23232323     0 
+....snip....
+```
+
+###### Invocation
+```
+java -Xmx8g -jar <path_to_hellbender_jar> CallSegments -T <target_file_tsv> -S <seg_file> -O <output_called_seg_file> -sample <sample_name> 
+```
+
+#### Create PoN workflow
+
+This workflow can take some time to run depending on how many samples are going into your PoN and the number of targets you are covering.
+
+##### Additional requirements
+- Normal sample bam files to be used in the PoN.
+
+
+##### Overview of steps
+
+- Step 1. Collect proportional coverage
+- Step 2. Combine proportional coverage files
+- Step 3. Create the PoN file
+
+##### Step 1. Collect proportional coverage on each bam file
+
+This is exactly the same as the case sample workflow, except that this needs to be run once for each input bam file, each with a different output file name.  Otherwise, the inputs should be the same for each bam file.
+
+Please see documentation [above](#step-1-collect-proportional-coverage).
+
+IMPORTANT NOTE: You must create a list of the proportional coverage files (i.e. output files) that you create in this step.  One output file per line in a text file (see step 2)
+
+##### Step 2. Merge proportional coverage files
+
+This step merges the proportional coverage files into one large file with a separate column for each samples.
+
+###### Inputs
+- list of proportional coverage files generated (possibly manually) in step 1.  This is a text file.
+```
+/path/to/pcov_file1.txt
+/path/to/pcov_file2.txt
+/path/to/pcov_file3.txt
+....snip....
+```
+
+###### Outputs
+- merged tsv of proportional coverage 
+```
+CONTIG  START   END     NAME    SAMPLE1    SAMPLE2 SAMPLE3 ....snip....
+1       12191   12227   target1    8.835E-6  1.451E-5     1.221E-5    ....snip....
+1       12596   12721   target2    1.602E-5  1.534E-5     1.318E-5   ....snip....
+....snip....
+```
+
+###### Invocation
+```
+java -Xmx8g -jar  <path_to_hellbender_jar> CombineReadCounts --inputList <text_file_list_of_proportional_coverage_files> \
+    -O <output_merged_file> -MOF 200 
```

Comment:
Maximum open files.  I do not think that we want to have users fiddling with this and 200 should be fine.